### PR TITLE
use markdown for notebooks

### PR DESCRIPTION
**Pull request by szha**
## Description ##
use markdown for notebooks

### Essentials ###
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [x] enable notedown
- [x] convert notebooks to markdown format

### CODE REVIEW

**Comment by mli** [comment]
Job PR-166/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/3/index.html

**Comment by mli** [comment]
Job PR-166/8 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/8/index.html

**Comment by mli** [comment]
Job PR-166/10 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/10/index.html

**Comment by mli** [comment]
Job PR-166/12 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/12/index.html

**Comment by leezu** [comment]
It seems the syntax highlighting is still broken

**Comment by mli** [comment]
Job PR-166/15 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/15/index.html

**Comment by mli** [comment]
Job PR-166/16 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/16/index.html

File: Jenkinsfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ stage("Sanity Check") {
     ws('workspace/gluon-nlp-lint') {
       checkout scm
       sh """#!/bin/bash
+      git clean -f -d -x
```
Comment (leezu) [inline Jenkinsfile]: What about adding `--exclude=<pattern> ` to `git clean` to sapare the `tests/externaldata` folder from being deleted every time?

**Comment by mli** [comment]
Job PR-166/19 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/19/index.html

**Comment by mli** [comment]
Job PR-166/38 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/38/index.html

**Comment by szha** [comment]
Seems that the output cells are still broken.

**Comment by mli** [comment]
Job PR-166/41 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/41/index.html

**Comment by mli** [comment]
Job PR-166/45 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/45/index.html

**Comment by mli** [comment]
Job PR-166/50 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/50/index.html

**Comment by mli** [comment]
Job PR-166/51 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/51/index.html

**Comment by mli** [comment]
Job PR-166/53 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/53/index.html

**Comment by mli** [comment]
Job PR-166/54 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-166/54/index.html

**Pull request merged**